it/PostCustomerReadTimeISpec.scala

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import java.io.File
+
+import org.scalatest.DoNotDiscover
+import play.api.http.Status.{ BAD_REQUEST, CREATED }
+import play.api.http.{ ContentTypes, HeaderNames }
+
+@DoNotDiscover
+@SuppressWarnings(Array("org.wartremover.warts.All"))
+class PostCustomerReadTimeISpec extends ISpec {
+
+  "A POST request to /secure-messaging/conversation/{client}/{conversationId}/read-time" should {
+
+    "return CREATED when read time added to DB" in {
+      createConversation
```

Comment:
This is the only dodgy looking thing. Nullary methods that cause mutation should use `()` by convention in Scala. Can this not be a chain future or something, so the action can be controlled? Not super important, can merge without ultimately.

Reply:
Oh, I think you have put your finger on one of the problems I was having with these tests (intermittent failure) @alexflav23 Have updated. Thank you :)